apis/apps/v1beta3/emqxbroker_types.go

```diff
@@ -22,6 +22,10 @@ import (
 )
 
 type EmqxBrokerTemplate struct {
+	// If the Registry value is set,
+	// it will be used as the registry of emqx and emqx reloader,
+	// but this value will not be associated with the sidecar container
```

Comment:
Registry will used for EMQX owner image, like `${registry}/emqx/emqx:4.4.11` and `${registry}/emqx/emqx-operator-reloader`,  but it will not be use other images, like sidecar container or else.

Reply:
done